tsconfig.json

```diff
@@ -14,7 +14,9 @@
     },
     "files": [
         "types/index.d.ts",
-        "ts-tests/monday-sdk-js-module.test.ts",
-        "ts-tests/monday-sdk-js-global.test.ts"
+        "types/client-api.interface.ts",
+        "types/client-data.interface.ts",
+        "types/client-execute.interface.ts",
```

Comment:
these are already imported by `types/index.d.ts` so no need to add them

Reply:
Done – can be merged